lib/senders/index.js

```diff
@@ -8,7 +8,7 @@ var P = require('../promise')
 // This indirection exists to accommodate different config properties
 // in the old auth mailer. If/when the two config files are merged and
 // there's nothing left that imports mailer/config, it is safe to merge
-// raw.js and this file into one. Be careful not to mix the arguments
+// legacy_index.js and this file into one. Be careful not to mix the args
```

Comment:
I wrote this comment before I changed the name of that file in #1676 but only just spotted the mismatch, sorry.